Xamarin.Forms.Platform.Android/Extensions/NativeBindingExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Android.Widget;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public static class NativeBindingExtensions
+	{
+		public static void SetBinding(this AView self, Expression<Func<object>> memberLamda, Binding binding)
```

Comment:
yeah that was to avoid the linker removing it.. 
we will have to work a strategy for the linker.. I m thinking in a class that could be generated or the users would just fill with the necessary code.. With all the properties so the linker can include them.